src/native/fs/fs_napi.cpp

```diff
@@ -287,6 +323,62 @@ struct Stat : public FSWorker
     }
 };
 
+/**
+ * IsEncrypted is an spectrumscale op
+ */
+struct IsEncrypted : public FSWorker
```

Comment:
I would add this to the general purpose getxattr methods.
Sync with @romayalon who is planning to enhance those functions.